### PR TITLE
[Windows] Clarify requirement for native WinVFS

### PR DESCRIPTION
**Pull request by michaelstingl**
According to my knowledge, native WinVFS works on all Win versions > 1709 (Win 10, 11, Win server 2019 + 2022)

@TheOneRing works for you?

Backport to master and 2.9

### CODE REVIEW

**Comment by d7oc** [comment]
Is 1709 really a version you can find in any Windows variant? I looked at `winver` in Windows Server 2022 which looks like the following:
Screenshot 2023-01-11 at 14 16 42
Does 1709 then corresponding with the OX build number?

**Comment by michaelstingl** [comment]
See https://learn.microsoft.com/en-us/windows/win32/cfapi/cloud-files-api-portal

> Starting in Windows 10, version 1709, Windows provides the cloud files API.

**Comment by d7oc** [comment]
Ok but Windows 10 is a client version how do the server versions come into the game. I mean it's quite obvious that Windows 11 is > Windows 10 so for the client side this is clear, but how about Windows Server 2012?

**Comment by michaelstingl** [comment]
Windows Server 2016: version 1607
Windows Server 2019: version 1809

Did Microsoft abandon this with Windows Server 2022?
(https://learn.microsoft.com/en-us/windows-server/get-started/windows-server-release-info)

File: modules/ROOT/pages/installing.adoc [hunk 1]
```diff
@@ -31,7 +31,7 @@ Depending on the operating system used, some minimum system requirements need to
 Windows::
 * Windows 7+
 ** x86 with 32-bit or x86-64 with 64-bit
-** Native WinVFS available for Windows 10 version 1709 or later
+** Native WinVFS available for Windows versions 1709 or later
```
Comment (d7oc) [inline modules/ROOT/pages/installing.adoc:34]:
```suggestion
** Native WinVFS available for Windows versions 1709 or later (on Windows Server you find the version in `winver` in the second row, the known versions are: https://en.wikipedia.org/wiki/Windows_Server#Semi-annual_releases_(discontinued) )
```

**Comment by michaelstingl** [comment]
@EParzefall @mmattel could you help to make this easily consumable?

File: modules/ROOT/pages/installing.adoc (hunk 1 above)
Comment (mmattel) [reply modules/ROOT/pages/installing.adoc:34]: @EParzefall I have updated the suggestable commit from @d7oc, can you have a look if ok and commit if so, I will then check for more possible improvements in the document.

**Review by EParzefall — APPROVED** [review]

**Comment by mmattel** [comment]
I will check the page asap for more possble improvements regarding winvfs.

**Comment by voroyam** [comment]
Please add actual Versions like @michaelstingl did in the first post in brackets. No one knows what "1709" is. Our docs should be, in my view, a reference guide that should give you the info you need in clear words.

This is my suggestion: 

```
For Windows Server 2019 and 2022 (and versions above) are supported.
```

**Review by voroyam — DISMISSED** [review]
I would change

```
For Windows Server you find the version in `winver` in the second row. The known versions can be identified via the following referenced list: https://en.wikipedia.org/wiki/Windows_Server#Semi-annual_releases_(discontinued)[Semi-annual releases (discontinued)].
```

to 

For Windows Server 2019 and 2022 (and versions above) are supported.

**Comment by d7oc** [comment]
> I would change
> 
> ```
> For Windows Server you find the version in `winver` in the second row. The known versions can be identified via the following referenced list: https://en.wikipedia.org/wiki/Windows_Server#Semi-annual_releases_(discontinued)[Semi-annual releases (discontinued)].
> ```
> 
> to
> 
> For Windows Server 2019 and 2022 (and versions above) are supported.

If it would be that easy. The point is if I got it correctly: there is no fixed Windows Server edition (2019/2022...) which has included that. It was added during the lifespan of Windows Server 2019/2022 with a specific version. So you can't say that Windows Server XXXX is supported as it was included in version yyyy of Windows Server XXXX.

**Comment by voroyam** [comment]
> Looking again at the version / edition combinations I think that from the history only Windows Server 2016 and below should be affected by the version mess. Windows Server 2019 and above should have it from the beginning (to be confirmed).

confirmed by testing or more research? or both

**Comment by d7oc** [comment]
> > Looking again at the version / edition combinations I think that from the history only Windows Server 2016 and below should be affected by the version mess. Windows Server 2019 and above should have it from the beginning (to be confirmed).
> 
> confirmed by testing or more research? or both

Hopefully @michaelstingl can confirm this. If not we need to do more research or testing.

**Comment by michaelstingl** [comment]
Win Server not tested. 1709+ is the information from the developer docs (see https://github.com/owncloud/docs-client-desktop/pull/317#issuecomment-1378743291). Win Server builds are based on equivalent Win10/11 builds. I also wouldn't state Win Server is "supported" if we don''t test it. I can only say, Win Server builds based on 1709+ Windows come with the system API that's required for the native VFS.

**Comment by d7oc** [comment]
In this case we should really play it save and keep Windows Server completely out of the game.

**Comment by michaelstingl** [comment]
We could add a note to the FAQ page, to make it less official

**Comment by JRundfeldt** [comment]
I agree with @d7oc. Lets focus on normal Windows versions. If customers have success with using our client on Windows Server Versions, I am happy for them, but I dont think we should extend our supported systems in this moment.
I also do not think that we need to change the documentation - what is mentioned in the docs, is supported. What is not mentioned, is experimental and is not supported by us.

**Comment by voroyam** [comment]
> I agree with @d7oc. Lets focus on normal Windows versions. If customers have success with using our client on Windows Server Versions, I am happy for them, but I dont think we should extend our supported systems in this moment.
> I also do not think that we need to change the documentation - what is mentioned in the docs, is supported. What is not mentioned, is experimental and is not supported by us.

If @michaelstingl states that "we" don't want to test windows servers then we should at least document that windows servers are not supported and should be used on ones own risk or something like that.  @mmattel please find a fitting wording.

Customers, especially LDBV, are always trying something new and if they don't find a definitive statement in the docs they open tickets and then this whole thing here has to be repeated again and again. Email, SF Issue, GH issue , Doc Issue.

The whole purpose of the documentation is to clarify the features or a product, not keep things "secret" because we don't want to test stuff and just omit them from the docs.

**Comment by mmattel** [comment]
@michaelstingl I have updated the PR to make the information with regards to Windows server available but in a way where there is no guarantee or claim for support. In addition, I harmonized the content with regards to vfs a bit for better readbility. Pls comment so we can proceed.

**Review by voroyam — APPROVED** [review]

**Comment by michaelstingl** [comment]
Works for me 👍  but I can't approve my own PR

**Pull request merged**